Sources/PackageGraph/RepositoryPackageContainerProvider.swift

```diff
@@ -49,6 +47,21 @@ public class RepositoryPackageContainerProvider: PackageContainerProvider {
         // Create the container wrapper.
         return RepositoryPackageContainer(identifier: identifier, repository: repository, manifestLoader: manifestLoader)
     }
+
+    public func getContainer(for identifier: Container.Identifier, completion: @escaping (Result<Container, AnyError>) -> Void) {
+        // Uncomment the following lines to enable parallel cloning in the resolver.
+        //
+        // repositoryManager.lookup(repository: identifier) { result in
+        //     do {
+        //         let handle = try result.dematerialize()
+        //         let repository = try handle.open()
+        //         let container = RepositoryPackageContainer(identifier: identifier, repository: repository, manifestLoader: self.manifestLoader)
+        //         completion(Result(container))
+        //     } catch {
+        //         completion(Result(AnyError(error)))
+        //     }
+        // }
```

Comment:
We shouldn't check in commented code... I would rather check in the implementation and tests for it, but not enable the part in the resolver that actually starts using it.

In any case, this feature should probably start out with a flag to enable it (or disable it), and so we can eventually implement it but off by default, then switch the default around.